lib/sidekiq_unique_jobs/timeout_calculator.rb

```diff
@@ -1,34 +1,21 @@
 module SidekiqUniqueJobs
-  class TimeoutCalculator
```

Comment:
I would honestly prefer inheritance over composition in this case because you could clean up the child calculators a bit.  Do you see it?